src/pages-and-resources/discussions/DiscussionsSettings.jsx

```diff
@@ -51,9 +49,7 @@ function DiscussionsSettings({ courseId, intl }) {
     incomplete: false,
   },
   {
-    label: intl.formatMessage(messages.configureApp, {
-      name: app ? app.name : 'discussions',
-    }),
+    label: intl.formatMessage(messages.settings),
```

Comment:
quick question -- Is this `intl` is 3rd party library or being used from frontend-platform? 

Where can I read more about this?

Reply:
It's a third party library that we wrap with frontend-platform/i18n:

https://edx.github.io/frontend-platform/module-Internationalization.html